Manifests/ManifestsApple/com.apple.security.firewall.plist

```diff
@@ -220,6 +220,40 @@
 			<key>pfm_value_import_processor</key>
 			<string>com.apple.security.firewall.Applications</string>
 		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_macos_min</key>
+			<string>12</string>
```

Comment:
Elsewhere you use `12.0` for OS version.  I like being explicit so let's do that here.

Reply:
See c0ed346.